test/utils/utils.go

```diff
@@ -14,38 +16,39 @@ import (
 
 type TestCmd struct {
 	Cmd      []string
-	Stdout   string
+	Stdout   string // fakexec doesn't leverage stderr in CombinedOutput, so use stdout for stderr too
 	ExitCode int
 }
 
-func GetFakeExecWithScripts(calls []TestCmd) (*fakeexec.FakeExec, *fakeexec.FakeCmd) {
-	fexec := &fakeexec.FakeExec{ExactOrder: false, DisableScripts: false}
+func GetFakeExecWithScripts(calls []TestCmd) *fakeexec.FakeExec {
+	fexec := &fakeexec.FakeExec{ExactOrder: true, DisableScripts: false}
 
 	fcmd := &fakeexec.FakeCmd{}
 
 	for _, call := range calls {
 		stdout := call.Stdout
+		ccmd := call.Cmd
 		if call.ExitCode != 0 {
 			err := &fakeexec.FakeExitError{Status: call.ExitCode}
-			fcmd.CombinedOutputScript = append(fcmd.CombinedOutputScript, func() ([]byte, []byte, error) { return []byte(stdout), nil, err })
+			fcmd.CombinedOutputScript = append(fcmd.CombinedOutputScript, func() ([]byte, []byte, error) { return []byte(stdout), []byte{}, err })
```

Comment:
Curious - where []byte{} is used and what is purpose of it? 
It seems fakeexec does not even use this return value ([]byte{}).

Reply:
this is true, nil is set on line 35 and it should be nil here too